curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md

```diff
@@ -18,15 +18,15 @@ localeTitle: Use seletores CSS para elementos de estilo
 ```yml
 tests:
   - text: Remova o atributo style do seu elemento <code>h2</code> .
-    testString: 'assert(!$("h2").attr("style"), "Remove the style attribute from your <code>h2</code> element.");'
+    testString: assert(!$("h2").attr("style"));
   - text: Crie um elemento de <code>style</code> .
-    testString: 'assert($("style") && $("style").length > 1, "Create a <code>style</code> element.");'
+    testString: assert($("style") && $("style").length >= 1);
   - text: Seu elemento <code>h2</code> deve ser azul.
-    testString: 'assert($("h2").css("color") === "rgb(0, 0, 255)", "Your <code>h2</code> element should be blue.");'
+    testString: assert($("h2").css("color") === "rgb(0, 0, 255)");
```

Comment:
I suggest the text translated
```suggestion
    testString: 'assert($("h2").css("color") === "rgb(0, 0, 255)", "Seu elemento <code>h2</code> deveria ser azul.");'
```

Reply:
We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.